src/telemetry/telemetry.c

```diff
@@ -400,6 +401,18 @@ build_telemetry_report()
 	ts_jsonb_add_str(parse_state,
 					 REQ_INSTALL_TIME,
 					 format_iso8601(ts_telemetry_metadata_get_install_timestamp()));
+	ts_jsonb_add_str(parse_state, REQ_INSTALL_METHOD, TIMESCALEDB_INSTALL_METHOD);
+
+	if (ts_version_get_os_info(&osinfo))
+	{
+		ts_jsonb_add_str(parse_state, REQ_OS, osinfo.sysname);
+		ts_jsonb_add_str(parse_state, REQ_OS_VERSION, osinfo.version);
+		ts_jsonb_add_str(parse_state, REQ_OS_RELEASE, osinfo.release);
+		if (osinfo.has_pretty_version)
```

Comment:
I guess we assume that it is ok for the receiver parser if some fields are missing?

Reply:
Yes, it should be OK. Also, this code was here before but removed by mistake. So, this is not new.